### PR TITLE
Add load balancer address to identity

### PR DESCRIPTION
**Pull request by tokle**

### CODE REVIEW

**Review by bjorncs — APPROVED** [review]

**Comment by bjorncs** [comment]
Merging now to test it out in CD asap

File: config-model/src/main/java/com/yahoo/vespa/model/container/Identity.java [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
 /**
  * @author mortent
  */
-public class Identity {
+public class Identity extends SimpleComponent implements IdentityConfig.Producer {
     private final String domain;
     private final String service;
 
     public Identity(String domain, String service) {
+        super("com.yahoo.container.jdisc.athenz.impl.AthenzIdentityProviderImpl");
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/Identity.java]: If the class is available on the classpath, use `com.yahoo.container.jdisc.athenz.impl.AthenzIdentityProviderImpl.class.getName()` instead

File: config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java [hunk 2]
```diff
@@ -881,4 +864,15 @@ public MbusParams(Double maxConcurrentFactor, Double documentExpansionFactor, In
         }
     }
 
+    public void setIdentity(Identity identity) {
+        this.identity = identity;
+        addComponent(identity);
+    }
+
+    @Override
+    public void getConfig(IdentityConfig.Builder builder) {
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java]: Why this routing of config? Just to allow your unit test to pass? Switch the test to use the real configId instead!

File: config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java [hunk 3]
```diff
@@ -881,4 +864,15 @@ public MbusParams(Double maxConcurrentFactor, Double documentExpansionFactor, In
         }
     }
 
+    public void setIdentity(Identity identity) {
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java]: This should not be necessary.

File: config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java [hunk 4]
```diff
@@ -881,4 +865,20 @@ public MbusParams(Double maxConcurrentFactor, Double documentExpansionFactor, In
...
+        this.identity = identity;
+        addComponent(identity);
+    }
+
+    @Override
+    public void getConfig(IdentityConfig.Builder builder) {
+        if (identity != null) {
```
Comment (gjoranv) [inline config-model/src/main/java/com/yahoo/vespa/model/container/ContainerCluster.java]: This code should be in the Indentity, not in ContainerCluster

File: container-core/src/main/resources/configdefinitions/identity.def [hunk 5]
```diff
@@ -2,3 +2,4 @@ namespace=container.core.identity
 
 domain string
 service string
+loadBalancerAddress string
```
Comment (gjoranv) [inline container-core/src/main/resources/configdefinitions/identity.def]: Please always insert trailing newline.

**Pull request merged**